docs/faq.md

```diff
@@ -172,11 +172,16 @@ new location [here]({{ site.baseurl }}/advanced-topics/corpora/#viewing-the-corp
 
 ## Does OSS-Fuzz support AFL or honggfuzz?
 
-OSS-Fuzz *uses* both [AFL](https://lcamtuf.coredump.cx/afl/) and
-[honggfuzz](https://github.com/google/honggfuzz)
-[fuzzing engines]({{ site.baseurl }}/reference/glossary/#fuzzing-engine).
-Follow the [ideal integration guide] and OSS-Fuzz will use all its fuzzing
-engines on your code.
+OSS-Fuzz *uses* the following
+[fuzzing engines]({{ site.baseurl }}/reference/glossary/#fuzzing-engine):
+
+1. [libFuzzer](https://llvm.org/docs/LibFuzzer.html).
+1. [AFL++](https://github.com/AFLplusplus/AFLplusplus), an improved and
```

Comment:
nit: 2. and 3.?

Reply:
in md format, just repeating 1. works, it is better as it does auto-bulleting.